ckanext/pages/plugin.py

```diff
@@ -48,9 +48,9 @@ def build_pages_nav_main(*args):
 
     for page in pages_list:
         if page['page_type'] == 'blog':
-            link = h.literal('<a href="/blog/%s">%s</a>' % (str(page['name']), str(page['title'])))
+            link = h.literal('<a href="/blog/%s">%s</a>' % (unicode(page['name']), unicode(page['title'])))
```

Comment:
I think the first value should be URL-escaped and the second needs to be HTML-escaped instead. AFAIK, `h.literal` by design does neither.